invoices/sql_store.go

```diff
@@ -25,9 +25,9 @@ const (
 	queryPaginationLimit = 100
 )
 
-// InvoiceQueries is an interface that defines the set of operations that can be
-// executed against the invoice database.
-type InvoiceQueries interface { //nolint:interfacebloat
+// SQLInvoiceQueries is an interface that defines the set of operations that can
+// be executed against the invoice SQL database.
+type SQLInvoiceQueries interface { //nolint:interfacebloat
```

Comment:
I wonder if this should instead live in `channeldb`? 

This why then the `invoice` package declares the interface, but doesn't necessarily implement it itself.

Reply:
My simple reasoning for also moving the interface here was that the SQL InvoiceDB implementation depends on this interface hence it's maybe best kept along with the implementation itself. If we move though perhaps `sqldb` might be a better choice?